### ISSUE TITLE
Can TESTNET be reset?

### ISSUE THREAD

**Issue by Admiral-Noisy-Bottom** [monero-project/monero#3458]
I'm having a problem with pool payments and it might be related to me using --stagenet. After downloading Linux binaries (I had previously built from source) I discovered that --stagenet isn't even an option.

I'm switching to --testnet to confirm that I'm not imagining the --stagenet issue.

BUT, the damned test net is almost as huge as the real net. FFS, now I have to wait days for it to sync up!!!!

There doesn't seem to be a bootstrap for testnet!!!!

Chances are I'm about to waist days syncing only to find my problem still exists.

CAN TESTNET BE RESET SO PEOPLE CAN TEST WITHOUT WAITING A LIFE TIME.

**Comment by moneromooo-monero** [comment]
No. However, now that we have network types, maybe a "instant start on version N" network might be a good idea.

**Comment by Admiral-Noisy-Bottom** [comment]
Oh, I looked at the --offline option and ran monerod --offline for a short while.

Can my pool be tested like that? Without peers, will a genesis block get created and then further blocks for test purposes?

That would be awesome and I'll try it - if it works I'll retract my FFS comment.

**Comment by moneromooo-monero** [comment]
I certainly test stuff like that when I don't have a compatible daemon and don't need more than one node. A genesis block will get created if you're starting without one. If you have a partly synced chain, it'll just use that.

**Comment by Admiral-Noisy-Bottom** [comment]
It didn't work, so I've continued with the sync process.

Daemon started using the --testnet --offline and all looked normal.

Wallet created using --testnet and then opening in wallet-rpc using --testnet.

But it all turned to custard when the pool node started. It went round and round in circles with a message "Core busy".

Oh well, it was worth a try. My testnet is only 346 days behind - it'll get there eventually.

**Comment by moneromooo-monero** [comment]
What commit hash are you running ? Recent versions should not be doing that.

**Comment by Admiral-Noisy-Bottom** [comment]
I assume by commit hash you're referring to the version or something like that. I download a binary from your site yesterday. 

I had also download source and built my own binaries (it has --stagenet but the binaries downloaded yesterday don't).

But the one I'm running now is from your getmonero.org site for Linux 64.

**Comment by moneromooo-monero** [comment]
Then use the one you built, which should be recent enogh.

**Comment by Admiral-Noisy-Bottom** [comment]
Ok, I've made a discovery (I think). The pool was not paying while using --stagenet and the binaries I built. I download the pre-built binaries from your site and synced --testnet (--stagenet doesn't exist) and payment to miners works. In summary, miners not payed while on stagenet and binaries built from latest source. Miners are paid using binaries pre-built and testnet. Odd!

I changed nothing in the pool. The only thing that changed was the binaries and I moved to --testnet. I'm about to switch back to the binaries I built and use --testnet and see what happens.
